usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -917,7 +1359,6 @@ def test_load_table_to_from_delta_for_recipient_profile(
     verify_delta_table_loaded_from_delta(spark, "recipient_profile", jdbc_inserts=True, ignore_fields=["id"])
 
 
-@mark.skip
 @mark.django_db(transaction=True)
 def test_load_table_to_from_delta_for_transaction_fabs(spark, s3_unittest_data_bucket, hive_unittest_metastore_db):
```

Comment:
I think this along with the tests for loading `transaction_normalized` and `transaction_fabs` can be removed. We are no longer loading these tables to delta.

Reply:
okay cool